bin/merge-configs.js

```diff
@@ -91,11 +91,30 @@ module.exports = (userConfig) => {
   } else {
     combinedWebpackConfig.resolve.alias.middleware = path.resolve(__dirname, '../lib/helpers/empty.js');
   }
-  if (universalReduxConfig.rootComponent) {
-    combinedWebpackConfig.resolve.alias.rootComponent = universalReduxConfig.rootComponent;
-  } else {
-    combinedWebpackConfig.resolve.alias.rootComponent = path.resolve(__dirname, '../lib/helpers/rootComponent.js');
-  }
+  // Try to resolve the configured plugins
+  const plugins = universalReduxConfig.plugins
+    .map((plugin) => {
+      const pluginPath = [
+        `${sourceDir}/${plugin}`,
+        `universal-redux-plugin-${plugin}`,
+        `${plugin}`
+      ].find((pp) => {
+        try {
+          require(pp);
```

Comment:
I don't think it does. It gets executed on the server and when webpack compiles the browser code.